experimenter/experimenter/experiments/api/v5/types.py

```diff
@@ -502,6 +502,9 @@ class NimbusExperimentType(DjangoObjectType):
     conclusion_recommendation = graphene.Field(
         NimbusExperimentConclusionRecommendationEnum
     )
+    conclusion_recommendations = graphene.List(
+        NimbusExperimentConclusionRecommendationEnum
+    )
```

Comment:
```
conclusion_recommendations = graphene.List(
    NimbusExperimentConclusionRecommendationEnum
)
```

generates 
```
conclusionRecommendations: (NimbusExperimentConclusionRecommendationEnum | null)[] | null;
```

We definitely dont want to accept `[null]` here, so we need to wrap the `NimbusExperimentConclusionRecommendationEnum` in `graphene.NonNull`

Do we want the field to accept null?  (probably not) If no, we need to wrap it in a nonnull.

```
graphene.NonNull(graphene.List(graphene.NonNull(NimbusExperimentConclusionRecommendationEnum)))
```
This will generate

```
conclusionRecommendations: NimbusExperimentConclusionRecommendationEnum[];
```

Reply:
Since in my model I have defined 
` conclusion_recommendations = models.JSONField(
        verbose_name="Conclusion Recommendations", blank=True, null=True
    )`
I think it should be ready to accept null, for example when we just create the experiment at that time it won't have anything

Reply:
or I need to change the model to provide default value `[]`

Reply:
yeah change the default to be [] and we dont have to worry about nulls anywhere.